### PR TITLE
Text annotator and redesign icons

### PR DESCRIPTION
**Pull request by vincerubinetti**
- replace custom tooltip directive with tippy (vue 3 version is still in alpha, which is probably why i didn't do this from the beginning, but it still seems to work fine so far)
- redesign/rework category icons in figma to make more consistent and only use stroke (no fill) and use `currentColor`
- upload latest backup of figma sketch file
- add options to icon component to add simple circular border around icon (for category icons)
- remove debounce from persist to avoid bug
- get rid of custom directive files (there should be no need for them besides a tooltip)
- add api call to text annotator biolink endpoint and parse/format results
- create upload button component that can do file uploads by browse or drag & drop
- css tweaks
- add util func to download json
- add text annotator HOC that shows results with tooltips, links, download button, etc, and has multiple upload methods including example text
- add tests

### CODE REVIEW

**Comment by vincerubinetti** [comment]
@putmantime Now would be a good time to ask if we want to include any other options that the endpoint provides:

image

Currently, in ui 2.0, there's only a checkbox for `longest_only`.

**Comment by vincerubinetti** [comment]
@putmantime @falquaddoomi

File: src/api/text-annotator.ts [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+interface Response {
+  content: string;
+  spans: Array<{
+    start: number;
+    end: number;
+    text: string;
+    token: Array<{
+      id: string;
+      category: Array<string>;
```
Comment (falquaddoomi) [inline src/api/text-annotator.ts:11]: Just FYI, this should change to `categories` if my proposed fix for biolink is accepted.

**Review by falquaddoomi — APPROVED** [review]
Looks good to me! I left a note about the `category` field name changing, but I don't think it should block merging this PR.

File: src/api/text-annotator.ts (hunk 1 above)
Comment (vincerubinetti) [reply src/api/text-annotator.ts:11]: Can you tag me in that PR so I know when it's merged

**Pull request merged**